lib/commands/base-bundler.ts

```diff
@@ -0,0 +1,14 @@
+export abstract class BundleBase {
+	constructor(protected $projectData: IProjectData,
+		protected $errors: IErrors,
+		protected $options: IOptions) {
+		this.$projectData.initializeProjectData();
+	}
+
+	protected validateBundling(): void {
+		const bundlerPluginName = "nativescript-dev-webpack";
```

Comment:
Can't we make this a bit more generic and use the option passed to --bundle to get the actual plugin from a collection?